asb-native/src/main/java/org/ballerinax/asb/sender/MessageSender.java

```diff
@@ -52,8 +57,7 @@
  */
 public class MessageSender {
 
-    private static final Logger log = Logger.getLogger(MessageSender.class);
-    private ServiceBusSenderClient sender;
+    private static final Logger LOGGER = Logger.getLogger(MessageSender.class);
```

Comment:
Any reason to define it as all caps? `LOGGER `

Reply:
It seems to be the common convention when naming static and final variables (which can be considered as constants)